src/helpers/passwordGenerator.js

```diff
@@ -0,0 +1,52 @@
+export const passwordGenerator = (options) => {
+    // This method should not be used to create safe/permament passwords for users
+    // Passwords should be generated by browsers or password managers
+    // Only justifiable use-case is creating temporary passwords valid until first login
+    const uppercase = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ';
+    const lowercase = uppercase.toLowerCase();
+    const numbers = '0123456789';
+    const specialCharacters = '!@#$%^&*()_+-=[]{}|;:,.<>?';
```

Comment:
You can consider moving it to constants. But not obligatory since it is used just here.

Reply:
I would prefer to keep it self-contained in that case.